openscap_report/scap_results_parser/parsers/oval_result_parser.py

```diff
@@ -30,7 +30,7 @@ def _get_oval_reports(self):
         oval_reports = {}
         reports = self.root.find('.//arf:reports', NAMESPACES)
         if reports is None:
-            raise MissingOVALResult
+            raise MissingOVALResult("all_OVALs")
```

Comment:
why "all_OVALs"?

Reply:
Because there should be an ID of the OVAL result that is missing. But in this case, there are missing all OVAL results.